README.md

```diff
@@ -1,237 +1,61 @@
-# Class Extensions and Inheritance
+# Class Extensions and Inheritance: `extends`
 
 ## Learning Goals
 
 - Use the `extends` keyword
-- Use the `super` method
-- Explain when `class` inheritance should be used in JavaScript
 
 ## Introduction
 
 In JavaScript, as in other Object Oriented languages, we've learned
-that you can create `class`es and build methods that can perform
-actions on instance data, or specific to the `class`. What if you have
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
+that you can create classes and build methods that can perform
+actions on instance data, or specific to the class. What if you have
+classes that exhibit many of the same behaviors, such as `Cat`, `Dog`,
 and `Bird`, which all have a method for `speak`? In JavaScript, you can
-create "child" object `class`es that inherit methods from their "parent"
-`class`es. In this lesson, we'll discuss 2 ways of _extending_ functionality
-to other `class`es.
+create "child" object classes that inherit methods from their "parent"
+classes. In this lesson, we'll discuss 2 ways of _extending_ functionality
+to other classes.
 
-## Use the `extend` Keyword
+## Use the `extends` Keyword
 
-To get started with inheriting `class` functionality, we utilize the `extends`
-keyword. `extends` is used in `class` declarations to create a `class` which
-is a _child_ of another `class`. 
+To get started with inheriting class functionality, we utilize the `extends`
+keyword. `extends` is used in class declarations to create a class which
+is a _child_ of another class. 
 
 ```js
 class Pet {
   constructor(name) {
     this.name = name;
   }
-}
 
-// Inherits from Pet
-class Dog extends Pet {
   speak() {
```

Comment:
I've updated some the content on method types and getters/setters. The suggestion I've added is to use getters and setters whenever a class is dealing with input or output (get and set define the "public menu" of a class). 

If we can, let's stick to using getters and setters in examples. Class methods are sort of downgraded to only function internally as helper methods in this design.